.nf-core.yml

```diff
@@ -5,15 +5,16 @@ lint:
     - assets/email_template.txt
     - lib/NfcoreTemplate.groovy
     - assets/multiqc_config.yaml
+  files_exist:
+    - bin/scrape_software_versions.py
+    - modules/local/get_software_versions.nf
```

Comment:
I thought these were removed?

Reply:
Yep, but the linting will be performed relative to the latest tools release where they are still a requirement. Can remove them from there after the next release.